powerlevel9k.zsh-theme

```diff
@@ -578,30 +578,39 @@ prompt_ip() {
 prompt_load() {
   # The load segment can have three different states
   local current_state="unknown"
+  local cores
+
+  if [[ "$OS" == "OSX" ]]; then
+    cores=$(sysctl -a | grep machdep.cpu)
```

Comment:
`sysctl` gives you the key and value by default. If you just want the value, use `sysctl -n`. And if you pass a key to it, there is no need to grep.
That being said, the key `machdep.cpu` gives me a whole bunch of information about my cpu, not just my amount of cores..
`sysctl -n hw.ncpu` is deprecated, and equivalent to `sysctl -n hw.logicalcpu`.
To get the amount of physical CPUs we could use either `sysctl -n hw.physicalcpu`.